src/components/Nav.js

```diff
@@ -9,11 +10,11 @@ function Nav() {
     return(
         <nav>
             <Container>
-                <div className="wrapper flexBox">
-                    <h2 className="headline navAnchor flexGrow_2">A. Bergsma</h2>
-                    <p className="navItem link">About</p>
-                    <p className="navItem link">Blog</p>
-                    <p className="navItem link">Contact</p>
+                <div className="flexBox">
```

Comment:
I've been really inconsistent with camelCasing and chain-casing in this project. I can't decide which one I prefer... and when I spend too much time with python I come over here and snake_case all over the place. Must select a style. I think I lean toward camelCasing, though that seems more JS specific, while chain-casing feels more natural in CSS.